other/require-unique-external-dns/svc.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    external-dns.alpha.kubernetes.io/hostname: "1.1.1.1"
```

Comment:
These aren't host names but that's ok, it still works.

Reply:
I'll change them 😅